core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -801,15 +801,21 @@ object ConsumerGroupCommand extends Logging {
         partitionsToReset.map { topicPartition =>
           logStartOffsets.get(topicPartition) match {
             case Some(LogOffsetResult.LogOffset(offset)) => (topicPartition, new OffsetAndMetadata(offset))
-            case _ => CommandLineUtils.printUsageAndDie(opts.parser, s"Error getting starting offset of topic partition: $topicPartition")
+            case _ => {
+              CommandLineUtils.printUsageAndDie(opts.parser, s"Error getting starting offset of topic partition: $topicPartition")
+              Exit.exit(1)
```

Comment:
Thanks @fvaleri . Wondering if these and other similar lines can be changed to use ToolsUtils. printUsageAndDie()?

Reply:
Yes, there were some some replacements that I missed. Fixed.